src/collections/blog/2025/01-01-meet-the-maintainer-sudhanshu-dasgupta/post.mdx

```diff
@@ -0,0 +1,229 @@
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Vivek:</span>
+  <p>
+    Have you worked with any other open source project? How does Layer5 compare?
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Sudhanshu:</span>
+  <p>
+     I have worked with a few other projects but they were not so large scale projects.
+     In comparison with Layer5, it is the community. Open source projects don't only mean how big the project is or how much you code. It is how engaged you are and about collaboration. This is the speciality of Layer5.
+     You will see meshmate and community managers helping though your way and getting success.  In other projects I don't know if you will get such one to one interactions often.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Vivek:</span>
+  <p>
+    <Link to="/projects">Layer5 projects</Link> have a number of active, open
+    source projects. You’ve been consistently contributing to a few of them.
+    Which one(s) are you currently focusing on?
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Sudhanshu:</span>
+  <p>
+    Ok so there are many I am focusing on. One of them is of course Meshery itself. We are building the feature of multi-tenancy of environments and workspaces. The second is Sistent which is Layer5's own design system to migrate all components to this sistent.
+    And the one is Layer5's own site which is running on gatsby so helping in getting prs reviewed there and merged and some dependency migration is also going on which i am focusing on.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Vivek:</span>
+  <p>What’s the coolest Meshery demo you have done/seen?</p>
+</div>
+
+<div class="interviewee">
+  <span>Sudhanshu:</span>
+  <p>
+    I think it could be one of the sessions hosted by Whitney Lee in her program on Infrastructure by Design with Meshery. So it was a very creative demo we saw there like she was noting stuff what Lee was sharing. So that was cool and one that actually happened recently was an event hosted by chatloopbackoff . One of the cncf ambassador came and just started using and explaining things from scratch. So that was also very informative. 
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Vivek:</span>
+  <p>What is your favorite feature in Meshery UI?</p>
+</div>
+
+<div class="interviewee">
+  <span>Sudhanshu:</span>
+  <p>
+     Ok yeah, my favorite feature is how you can import various existing designs like k8s manifest, helm, or docker compose though meshery's import modal. And boom when it gets imported and you can visualize that in kanvas which is one of the meshery's extensions and can do multiple things. So that is one of the core features and my favorite.
```

Comment:
Capitalize pronouns.

How about "drag and drop into Kanvas" instead of "meshery's import modal"?